internal/common/logs.go

```diff
@@ -9,42 +9,35 @@ import (
 	"io"
 	"log"
 	"os"
-	"sync"
 	"time"
 
 	"go.uber.org/multierr"
 )
 
 // StartLogWrite starts go routines to write logs to each of the logs that are monitored by CW Agent according to
 // the config provided
-func StartLogWrite(configFilePath string, agentRunDuration time.Duration, dataRate int) error {
-	//create wait group so main test thread waits for log writing to finish before stopping agent and collecting data
-	var (
-		multiErr error
-		wg       sync.WaitGroup
-	)
+func StartLogWrite(configFilePath string, duration time.Duration, logLinesPerMinute int) error {
+	var multiErr error
 
 	logPaths, err := getLogFilePaths(configFilePath)
 	if err != nil {
 		return err
 	}
 
 	for _, logPath := range logPaths {
```

Comment:
The [StartLogWrite is being used for generate logs for the validator](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/validator/validators/stress/stress_validator.go#L147-L148). However, we would only need to start it instead of using wait group so this would help us starting [LogWritng](https://github.com/aws/amazon-cloudwatch-agent-test/pull/166/files#diff-32ea2f52d0d817b42a874abb6e550c6c2040c34707954b8efd66e0c474660738R19) and [SendingMetrics](https://github.com/aws/amazon-cloudwatch-agent-test/pull/166/files#diff-781bb3810ede78771b4deb63bb6bb797a2d06434f0d844429b5ad16a20304e0fR14) concurrently without sequentially waiting for the [wait group](https://gobyexample.com/waitgroups) and having a sleep here to wait for the generator to finish [generate load](https://github.com/aws/amazon-cloudwatch-agent-test/pull/166/files#diff-36fa5ec31f40a4d9a878623ba1993272853ab2125e64152317da2a66cc7365d6R51)